core/arch/aarch64/instr_create.h

```diff
@@ -65,41 +65,30 @@
  */
 #define ISZ_DOUBLE 3
 
-/**
- * Used in an additional immediate source operand to a vector operation, denotes
- * half-precision floating point vector elements. See \ref sec_IR_AArch64.
- */
-#define FSZ_HALF 1
-
 /**
  * Operand indicating half-precision floating point vector elements for the
```

Comment:
A byte is not half-precision.  Also, are there single-byte floating-point elements?  This is supposed to be more general and cover integer elements, right?

Reply:
I've updated the comments, stating the width. That should be independent of integer/FP.